sample/android/app/src/debug/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
```

Comment:
I'm pretty sure this isn't needed. This permission is already in the main manifest.  When building a debug variant, this debug manifest is applied in addition to the main manifest.